libs/openchallenges/ui/src/lib/discord-button/discord-button.component.ts

```diff
@@ -0,0 +1,15 @@
+import { CommonModule } from '@angular/common';
+import { Component, Input } from '@angular/core';
+import { MatLegacyButtonModule as MatButtonModule } from '@angular/material/legacy-button';
+
+@Component({
+  selector: 'openchallenges-discord-button',
+  standalone: true,
+  imports: [CommonModule, MatButtonModule],
+  templateUrl: './discord-button.component.html',
+  styleUrls: ['./discord-button.component.scss'],
+})
+export class DiscordButtonComponent {
+  @Input({ required: false }) label = 'Discord';
```

Comment:
Could we use a more descriptive label here?  e.g. "Contact us on Discord"

Reply:
"Join us on Discord"? (a bit shorter - I will post a preview soon)